src/map/itemdb.c

```diff
@@ -1571,7 +1571,12 @@ static void itemdb_read_chains(void)
 				prev->next = &itemdb->chains[count].items[c - 1];
 
 			itemdb->chains[count].items[c - 1].id = data ? data->nameid : 0;
-			itemdb->chains[count].items[c - 1].rate = data ? libconfig->setting_get_int(entry) : 0;
+
+			int rate = data ? libconfig->setting_get_int(entry) : 0;
+			if (battle_config.item_rate_adddrop != 100)
+				itemdb->chains[count].items[c - 1].rate = rate*battle_config.item_rate_adddrop/100;
```

Comment:
missing spaces near ``*`` and near ``/``

Reply:
so I copied that [from here](https://github.com/HerculesWS/Hercules/blob/fbd9c8f9df8022619dcae158664bb5e81c7ae4bf/src/map/pc.c#L2163)

```c
rate = rate*battle_config.item_rate_adddrop/100;
```

is there a linter/formatter/other tool I can use to catch these ?

Reply:
many code in hercules with wrong style. if if line changed or added in commit it should use correct style
don't need search other style issues. only follow style in new/changed code

Reply:
thanks that makes sense.  for future prs though, are there any tools which will catch style issues like this?  like linters or formatters?

Reply:
no, only manual checking

Reply:
okay no problem, i'll make sure to check next time